.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: php
-sudo: false
-dist: xenial
+dist: bionic
```

Comment:
On d1695ed1c0d7ae0db1c612fc1811b721c34e1524 we changed the file `tests/travis/install-sqlsrv-dependencies.sh` to point use the version 16.04, shouldn't we bump that version up again?

Reply:
True but let's do it in a separate PR since it's unrelated to the nightly build.

Reply:
Alright, let get this merged in then :+1: